tez-dag/src/test/java/org/apache/tez/dag/app/dag/TestRootInputInitializerManager.java

```diff
@@ -66,8 +69,11 @@
 
   @Before
   public void setUp() throws Exception {
-    ExecutorService rawExecutor = Executors.newCachedThreadPool(new ThreadFactoryBuilder()
-            .setDaemon(true).setNameFormat("Test App Shared Pool - " + "#%d").build());
+    Configuration conf = new Configuration(true);
+    ExecutorService rawExecutor =
```

Comment:
This is not the way to test the proposed changes here. Test overrides the behavior that we want to test. So it passes regardless of the actual patch anyway.

Reply:
True, wanted to keep it simple but added the extra logic for DAGAppMaster initialization